packages/nx/src/command-line/show/project.ts

```diff
@@ -70,5 +72,4 @@ export async function showProjectHandler(
       }
     }
   }
-  await output.drain();
```

Comment:
This has the potential to miss some output, we added it to fix an issue. What is the reason for this being reverted?